signup-default.properties

```diff
@@ -2,7 +2,7 @@
 mosip.signup.supported.challenge-type=OTP
 mosip.signup.supported.challenge-format-types={'alpha-numeric'}
 mosip.signup.id-schema.version=0.2
-mosip.signup.identifier.regex=^\\+855[1-9]{8,9}$
+mosip.signup.identifier.regex=\\+855[1-9]\\d{7,8}
```

Comment:
Why is the ^ and $ removed?

Reply:
It copy it from signup-server for fix validating phone number contain 0.
Let me add it back.